procurement_not_group_purchase_line/README.rst

```diff
@@ -0,0 +1,18 @@
+Procurement not group purchase line
+===================================
+
+This module defined in the product category "boolean" field "Group in
+purchase".
+If this new field is checked, to execute the procurement acts in a standard 
+adding the amount on line purchase order, but if it is not checked, a new line
```

Comment:
s/on line purchase order/on existing line purchase order if the product and the UoM is the same.